gradle.properties

```diff
@@ -32,4 +32,7 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
Worth putting in each gradle build file?

Reply:
Isn't in applied to all modules since it's defined in project level gradle file?

Reply:
Yep - personal style thing.  No problem.